mappings/net/minecraft/entity/boss/dragon/EnderDragonEntity.mapping

```diff
@@ -15,6 +15,7 @@ CLASS net/minecraft/unmapped/C_bsmypcfq net/minecraft/entity/boss/dragon/EnderDr
 	FIELD f_jzbebsvn latestSegment I
 	FIELD f_kuwsuxxt DRAGON_DEATH_TIME_KEY Ljava/lang/String;
 	FIELD f_lasuowby yawAcceleration F
+	FIELD f_lcffcjzk MAX_GROWL_INTERVAL I
```

Comment:
Could you include a rename of the field `MAX_HEALTH` (f_vwueqmnd) -> `MIN_GROWL_INTERVAL` in this PR? It's incorrectly named, probably since it also happens to be the dragon's health (except as an int instead of a double)

Reply:
Yep will do

Reply:
Sorted